docs/tools/kompose/user-guide.md

```diff
@@ -14,45 +14,44 @@ redirect_from:
 
 `kompose` is a tool to help users who are familiar with `docker-compose` move to **Kubernetes**. `kompose` takes a Docker Compose file and translates it into Kubernetes resources.
 
+More information about Kompose can be found on the official [http://kompose.io](http://kompose.io/) site.
+
 `kompose` is a convenience tool to go from local Docker development to managing your application with Kubernetes. Transformation of the Docker Compose format to Kubernetes resources manifest may not be exact, but it helps tremendously when first deploying an application on Kubernetes.
 
 ## Use Case
 
 If you have a Docker Compose `docker-compose.yml` or a Docker Distributed Application Bundle `docker-compose-bundle.dab` file, you can convert it into Kubernetes deployments and services like this:
 
 ```console
-$ kompose --bundle docker-compose-bundle.dab convert
-WARN[0000]: Unsupported key networks - ignoring
-file "redis-svc.yaml" created
-file "web-svc.yaml" created
-file "web-deployment.yaml" created
-file "redis-deployment.yaml" created
-
 $ kompose -f docker-compose.yml convert
-WARN[0000]: Unsupported key networks - ignoring
+WARN: Unsupported key networks - ignoring
 file "redis-svc.yaml" created
 file "web-svc.yaml" created
 file "web-deployment.yaml" created
 file "redis-deployment.yaml" created
 ```
 
-## Installation 
+## Installation
+
+We have multiple ways to install Kompose. Our prefered method is downloading the binary from the latest GitHub release.
 
-Grab the latest [release](https://github.com/kubernetes-incubator/kompose/releases) for your OS, untar and extract the binary.
+### GitHub release
 
-### Linux
+Kompose is released via GitHub on a three-week cycle, you can see all current releases on the [GitHub release page](https://github.com/kubernetes-incubator/kompose/releases).
```

Comment:
This command would be outdated when a new release of kompose is out because it's hard-coded. (OTOH, Kubernetes release has https://storage.googleapis.com/kubernetes-release/release/stable.txt which always returns the latest stable release number). This seems obvious, but mentioning it ensures that users are aware of the release number being hard-coded into this `curl` command.